src/ui/interface.js

```diff
@@ -228,4 +230,67 @@ module.exports = function Interface(options) {
     $("[rel=popover]").popover()
     return true;
   });
+
+  //Start Handle multiple webcam resolutions
+  function changeResolution(w, h){
+    document.getElementById('image').setAttribute("width",w);
+    document.getElementById('image').setAttribute("height",h);
+  }
+  $('#qvga').click(function(e){
+    changeResolution('100px','100px')
+  });
+  $('#vga').click(function(e){
+    changeResolution('800px','600px')
+  });
+  $('#hd').click(function(e){
+    changeResolution('2400px','1800px')
+  })
+  $('#full-hd').click(function(e){
+    changeResolution('6000px','4000px')
+  });
+  //End Handling of Multiple webcam resolutions
+
+  //Start Canvas Recording and Download
+
+  const canvas = document.getElementById('image');
+  const ctx = canvas.getContext('2d');
+  var x = 0;
+  const stream = canvas.captureStream(); // grab our canvas MediaStream
+  const rec = new MediaRecorder(stream); 
```

Comment:
Does this cost any performance to start here, i.e. should we think about only setting this up if people start using video? Or, is it OK and not too much computational work?